pkg/testdata/test-cluster-identity.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this the data returned for a given shard (e.g. a business unit) ? 
If yes, then can we rename the test file to be something like: `test-cluster-identity-per-shard`

Reply:
no, this is the data for all the identities for a given cluster name